README.md

```diff
@@ -88,8 +88,7 @@ type BucketReader interface {
 	// IsObjNotFoundErr returns true if error means that object is not found. Relevant to Get operations.
 	IsObjNotFoundErr(err error) bool
 
-	// IsAccessDeniedErr returns true if access to object is denied.
-	IsAccessDeniedErr(err error) bool
+	// IsAccessDeniedErr returns true if acces to object is denied.
```

Comment:
This seems a bit off - do we need to edit the mdox line above?

Reply:
You're right, the method comment had a typo.